sonar-orchestrator/src/test/java/com/sonar/orchestrator/build/ScannerForMSBuildInstallerTest.java

```diff
@@ -75,7 +76,7 @@ public void install_zip() throws Exception {
 
     assertThat(script).isFile().exists();
     assertThat(script.getName()).contains("SonarScanner.MSBuild.exe");
-    assertThat(script.getParentFile()).hasName("sonar-scanner-msbuild");
+    assertThat(script.getParentFile()).hasName("sonar-scanner");
```

Comment:
When installed from a specified zip file, the scanner was added to a hardcoded folder. I've changed that folder to be `sonar-scanner`. In the documentation we use [SonarScanner for .NET](https://docs.sonarsource.com/sonarqube/9.8/analyzing-source-code/scanners/sonarscanner-for-dotnet/)